src/Events.Functions/Altinn.Platform.Events.Functions.csproj

```diff
@@ -11,12 +11,11 @@
     <PackageReference Include="Azure.Security.KeyVault.Certificates" Version="4.5.1" />
     <PackageReference Include="CloudNative.CloudEvents.AspNetCore" Version="2.6.0" />
     <PackageReference Include="CloudNative.CloudEvents.SystemTextJson" Version="2.6.0" />
-    <PackageReference Include="Microsoft.Extensions.Http" Version="7.0.0" />
+    <PackageReference Include="JWTCookieAuthentication" Version="3.0.0" />
```

Comment:
```suggestion
    <PackageReference Include="JWTCookieAuthentication" Version="3.0.1" />
```

Reply:
That didn't compile, but it might work now that I removed the direct dependency to the Token package.